src/components/calcite-action/calcite-action.tsx

```diff
@@ -43,7 +43,7 @@ export class CalciteAction {
   @Prop({ reflect: true }) active = false;
 
   /**
-   * Indicates the alignment when text-enabled is false.
+   * Sets the horizontal alignment of the button's content.
```

Comment:
Nitpick: since non-text actions are square, can we clarify this targets the text specifically? Something like "Optionally specify the horizontal alignment of button elements with text content."